crawl-ref/source/command.cc

```diff
@@ -596,6 +596,7 @@ void show_targeting_help()
     cols.add_formatted(1, targeting_help_2, true);
     show_keyhelp_menu(cols.formatted_lines());
 }
+
```

Comment:
Extra whitespace here. It's a minor thing, but better to have whitespace fixups in a separate commit specifically for that purpose.